### ISSUE TITLE
Problems with mteb_meta for german evaluation

### ISSUE THREAD

**Issue by achibb** [embeddings-benchmark/mteb#847]
Hi everyone, I am having troubles with generating the mteb_meta for German, with just running the script.

I am currently trying to format results but it does not seem to work straight away with "mteb_meta.py" - any idea? I just get a blank metadata file:

tags:

mteb
model-index:
name: gbert-large

results:

it gives this for every dataset:
WARNING:mteb.evaluation.MTEB:Passing task names as strings is deprecated and will be removed in the next release. Please use tasks = mteb.get_tasks(tasks=[...]) method to get tasks instead.
INFO:main:Skipping AmazonCounterfactualClassification as split test not present.
WARNING:mteb.evaluation.MTEB:Passing task names as strings is deprecated and will be removed in the next release. Please use tasks = mteb.get_tasks(tasks=[...]) method to get tasks instead.
INFO:main:Skipping AmazonReviewsClassification as split test not present.

Do I need to modify something on the code?

**Comment by imenelydiaker** [comment]
Due to recent updates #826 and #806:

If you're using a python script then your code should look like this:

```py
import mteb
from sentence_transformers import SentenceTransformer

# Define the sentence-transformers model name
model_name = "average_word_embeddings_komninos"
# or directly from huggingface:
# model_name = "sentence-transformers/all-MiniLM-L6-v2"

model = SentenceTransformer(model_name)
tasks = mteb.get_tasks(tasks=["Banking77Classification"])
evaluation = mteb.MTEB(tasks=tasks)
results = evaluation.run(model, output_folder=f"results/{model_name}")
```

**Comment by imenelydiaker** [comment]
> INFO:main:Skipping AmazonCounterfactualClassification as split test not present. 
> INFO:main:Skipping AmazonReviewsClassification as split test not present.

These look likely like a bug on our side, we'll check, thank you for reporting!

**Comment by KennethEnevoldsen** [comment]
@imenelydiaker I believe it is due to the new results format introduced in #759

**Comment by KennethEnevoldsen** [comment]
@achibb, we have currently updated the CLI and well as the benchmark lists. I believe the new CLI should suit your purpose

**Comment by achibb** [comment]
Thank you very much! Will test the next days and feedback.

I was wondering can I also compute something for the German benchmark for other models like mdeberta, and somehow add it to the leaderboard ?

Gesendet von Outlook für iOS<https://aka.ms/o0ukef>
________________________________
Von: Kenneth Enevoldsen ***@***.***>
Gesendet: Tuesday, June 11, 2024 1:33:36 PM
An: embeddings-benchmark/mteb ***@***.***>
Cc: achibb ***@***.***>; Mention ***@***.***>
Betreff: Re: [embeddings-benchmark/mteb] Problems with mteb_meta for german evaluation (Issue #847)


@achibb<https://github.com/achibb>, we have currently updated the CLI and well as the benchmark lists. I believe the new CLI should suit your purpose

**Comment by imenelydiaker** [comment]
> Thank you very much! Will test the next days and feedback. I was wondering can I also compute something for the German benchmark for other models like mdeberta, and somehow add it to the leaderboard ? 

Yes you can evaluate any model and submit results to [this repo](https://huggingface.co/datasets/mteb/results) via PR so they can be added to the leaderboard. (check guide on opening a PR on HF).
